nodechef.exp

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/expect -f
+#
+set timeout -1
+spawn $env(SHELL)
+match_max 100000
+#expect -exact "thouse@polter:~/Retrospectre\$ "
+send -- "cd src/deploy\r"
+#expect -exact "cd src/deploy\r
+#thouse@polter:~/Retrospectre/src/deploy\$ "
+send -- "bash nodechef\r"
+expect -exact "bash nodechef\r
+\[?\] Enter your email address and then press enter\r
+\[?\] Enter your password and then press enter\r
+"
+send -- "houset@msoe.edu\r"
+expect -exact "houset@msoe.edu\r
+"
+send -- "Zywave20!6\r"
```

Comment:
Work with John to get this to be a private Travis build variable, or else anyone else could just straight up deploy to our site. You can make the email private too, but up to you.